### PR TITLE
Adds the thing

### PR DESCRIPTION
**Pull request by tomcylke**
The maven structure in the repo isn't currently working so I didn't include a pom.xml, should still be fairly straightforward to compile.
Thank you Ganom and several others for doing most of the work on this plugin.

### CODE REVIEW

File: OneClick/src/main/java/net/runelite/client/plugins/oneclick/AlchItem.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package net.runelite.client.plugins.oneclick;
+
+import lombok.Getter;
+
+class AlchItem
```
Comment (Ganom) [inline OneClick/src/main/java/net/runelite/client/plugins/oneclick/AlchItem.java]: Getter annotation on the class

File: OneClick/src/main/java/net/runelite/client/plugins/oneclick/OneClickPlugin.java [hunk 2]
```diff
@@ -0,0 +1,547 @@
...
+	@Inject
+	private EventBus eventBus;
+
+	@Inject
+	private Client client;
+
+	@Inject
+	private OneClickConfig config;
+
+	final private Set<Integer> dartTips = new HashSet<>(Arrays.asList(ItemID.BRONZE_DART_TIP, ItemID.IRON_DART_TIP,
```
Comment (Ganom) [inline OneClick/src/main/java/net/runelite/client/plugins/oneclick/OneClickPlugin.java]: private static final Set<Integer> dartTips = ImmutableSet.of();

File: OneClick/src/main/java/net/runelite/client/plugins/oneclick/AlchItem.java (hunk 1 above)
Comment (tomcylke) [reply OneClick/src/main/java/net/runelite/client/plugins/oneclick/AlchItem.java]: Wtf IntelliJ I literally already had that https://i.imgur.com/GjxSjo0.png

File: OneClick/src/main/java/net/runelite/client/plugins/oneclick/OneClickPlugin.java [hunk 3]
```diff
@@ -0,0 +1,547 @@
...
+	}
+
+	private void onGameTick(GameTick event)
+	{
+		if (cannon != null)
+		{
+			final Entity entity = cannon.getEntity();
+			if (entity instanceof DynamicObject)
+			{
+				if (((DynamicObject) entity).getAnimationID() == 514 && prevCannonAnimation == 514)
```
Comment (Ganom) [inline OneClick/src/main/java/net/runelite/client/plugins/oneclick/OneClickPlugin.java]: final int anim = (DynamicObject) entity).getAnimationID()

File: OneClick/src/main/java/net/runelite/client/plugins/oneclick/OneClickPlugin.java [hunk 4]
```diff
@@ -0,0 +1,547 @@
...
+				if (i == 0)
+				{
+					menuList[i] = event.getMenuEntries()[i];
+				}
+				else
+				{
+					menuList[i + 1] = event.getMenuEntries()[i];
+				}
+			}
+			final MenuEntry SetHighAlchItem = new MenuEntry();
```
Comment (Ganom) [inline OneClick/src/main/java/net/runelite/client/plugins/oneclick/OneClickPlugin.java]: setHighAlchItem

**Pull request merged**